src/Phim/Color.php

```diff
@@ -1618,13 +1618,15 @@ public static function fade(ColorInterface $color, $ratio)
     }
 
     //http://www.easyrgb.com/index.php?X=DELT&H=05#text5
-    public static function getDifference(ColorInterface $color, ColorInterface $compareColor)
+    public static function getDifference(ColorInterface $color, ColorInterface $compareColor, $weights = [1,1,1])
```

Comment:
Can you typehint the $weights parameter as `array`? Then I'll merge it :)

Reply:
It might also be useful to make sure it's always 3 elements

`$weights = array_pad($weights, 3, 1);`

Reply:
Here are the changes.